.drone/drone.sh

```
@@ -0,0 +1,195 @@
+    echo 0 | sudo tee /proc/sys/kernel/randomize_va_space > /dev/null
+fi
+
+if [ "$DRONE_JOB_BUILDTYPE" == "boost" ]; then
+
+echo '==================================> INSTALL'
+
+common_install
+
+echo '==================================> SCRIPT'
+
+. $BOOST_ROOT/libs/$SELF/ci/build.sh
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)